flow/record/adapter/line.py

```diff
@@ -1,37 +1,66 @@
-from flow.record import open_path_or_stream
+from __future__ import annotations
+
+from functools import lru_cache
+
+from flow.record import RecordDescriptor, open_path_or_stream
 from flow.record.adapter import AbstractWriter
 from flow.record.utils import is_stdout
 
 __usage__ = """
 Line output format adapter (writer only)
 ---
-Write usage: rdump -w line://[PATH]
+Write usage: rdump -w line://[PATH]?verbose=[VERBOSE]
 [PATH]: path to file. Leave empty or "-" to output to stdout
+
+Optional arguments:
+    [VERBOSE]: Also show fieldtype in line output (default: False)
 """
 
 
+@lru_cache(maxsize=1024)
+def field_types_for_record_descriptor(desc: RecordDescriptor) -> dict[str, str]:
```

Comment:
Maybe nice to also add annotations to the rest of the file?

Reply:
Done